source/extensions/filters/http/ext_proc/ext_proc.cc

```diff
@@ -612,7 +620,7 @@ void Filter::onGrpcError(Grpc::Status::GrpcStatus status) {
     closeStream();
     // Since the stream failed, there is no need to handle timeouts, so
     // make sure that they do not fire now.
-    onFinishProcessorCalls();
+    onFinishProcessorCalls(absl::UnknownError(""));
```

Comment:
It would be good to translate the GrpcStatus 'status' value into an absl::Status and pass it into onFinishProcessorCalls here.

Reply:
We can pass GrpcStatus instead of absl::Status, this status is only for grpc call result anyway.

Reply:
SGTM!